README.md

```diff
@@ -132,6 +133,24 @@ Releases to PyPI is an Github Action which needs to be manually triggered.
 [![Release](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml/badge.svg)](https://github.com/spetlr-org/spetlr/actions/workflows/release.yml)
 [![PyPI spetlr badge](https://img.shields.io/pypi/v/spetlr)](https://pypi.org/project/spetlr/)
 
+
+# Requirements and dependencies
+The library has three txt-files at the root of the repo. These files defines three levels of requirements:
+- `requirements_install.txt` - this file contains the required libraries to be able to install spetlr.
+- `test_requirements.txt` - libraries required to run unit- and integration tests
```

Comment:
This file naming convention `test_requirements.txt` conflicts with standard `pytest` discoverability, which looks for files with a preceding `test*`-pattern in their name. Hence running pytest in the root will yield an error unless specifically set to ignore, but I would much rather just change the filename.

Reply:
I will rename